src/app/GitUI/CommandsDialogs/BrowseDialog/FormUpdates.Designer.cs

```diff
@@ -116,34 +117,46 @@ private void InitializeComponent()
             tlpnlContent.ColumnStyles.Add(new ColumnStyle());
             tlpnlContent.Controls.Add(UpdateLabel, 0, 0);
             tlpnlContent.Controls.Add(progressBar1, 0, 1);
-            tlpnlContent.Controls.Add(linkChangeLog, 0, 2);
+            tlpnlContent.Controls.Add(linkRequiredNetRuntime, 0, 3);
+            tlpnlContent.Controls.Add(linkChangeLog, 0, 4);
             tlpnlContent.Dock = DockStyle.Fill;
             tlpnlContent.Location = new Point(12, 12);
             tlpnlContent.Margin = new Padding(0);
             tlpnlContent.Name = "tlpnlContent";
-            tlpnlContent.RowCount = 4;
+            tlpnlContent.RowCount = 6;
             tlpnlContent.RowStyles.Add(new RowStyle());
             tlpnlContent.RowStyles.Add(new RowStyle());
+            tlpnlContent.RowStyles.Add(new RowStyle(SizeType.Absolute, 8F));
+            tlpnlContent.RowStyles.Add(new RowStyle());
             tlpnlContent.RowStyles.Add(new RowStyle());
             tlpnlContent.RowStyles.Add(new RowStyle(SizeType.Percent, 100F));
-            tlpnlContent.Size = new Size(432, 49);
+            tlpnlContent.Size = new Size(438, 67);
             tlpnlContent.TabIndex = 3;
             // 
+            // linkRequiredNetRuntime
+            // 
+            linkRequiredNetRuntime.AutoSize = true;
+            linkRequiredNetRuntime.Dock = DockStyle.Fill;
+            linkRequiredNetRuntime.LinkArea = new LinkArea(0, 0);
+            linkRequiredNetRuntime.Location = new Point(3, 49);
+            linkRequiredNetRuntime.Name = "linkRequiredNetRuntime";
+            linkRequiredNetRuntime.Size = new Size(432, 15);
+            linkRequiredNetRuntime.TabIndex = 2;
+            linkRequiredNetRuntime.Text = "Required: .NET {0} Desktop Runtime {1} or later {2}.x";
```

Comment:
`"Required: .NET 8.10 Desktop Runtime 8.10.134 or later 8.x"` from testcases is a good example why this unclear.
```suggestion
            linkRequiredNetRuntime.Text = "Required: .NET {0} Desktop Runtime {1} or later {0}.x";
```

Reply:
The suggestion is technically incorrect - not only 8.10.200 willwork, but 8.20.4 will work too.

It's quite unlikely there will be a release of X.1 of .NET (3.1 was an exception), so the whole discussion is mute, IMO.

Reply:
The default roll forward policy is latestFeature so 9.0.x will be supported. It is possible to set latestMinor or even latestMajor, but such forward compatibility cannot be guaranteed.
But I would prefer myself to just see the text before or later, the rest is distracting and should be in a link or so. But some users are disagreeing....
Later, go with this for now.